packages/@jsii/kernel/test/kernel.test.ts

```diff
@@ -79,7 +79,7 @@ defineTest('deleteObject will remove the reference', (sandbox) => {
 });
 
 defineTest('in/out primitive types', (sandbox) => {
-  const alltypes = sandbox.create({ fqn: 'jsii-calc.AllTypes', args: [] });
+  const alltypes = sandbox.create({ fqn: 'jsii-calc.compliance.AllTypes', args: [] });
```

Comment:
Didn't you remove the submodules from jsii-calc?

Reply:
This is a namespace that was _already_ there.

Reply:
This explains why tests won't pass :)